modules/phase_field/include/actions/GrainGrowthAction.h

```diff
@@ -24,9 +24,8 @@ class GrainGrowthAction : public Action
   virtual void act();
 
 protected:
-  void addKernel(const std::string & kernel_type,
-                 const std::string & kernel_name,
-                 InputParameters params);
```

Comment:
Why is this being removed?

Reply:
Hmm, and while you're at it would you mind making this virtual. I'm puzzled how this ever worked without being virtual...

Reply:
actually scratch that